files/usr/etc/profile.d/lang.sh

```diff
@@ -7,15 +7,28 @@
 #     $HOME/.i18n
 #
 
+unset _save
+
 #
-# Already done by the remote SSH side
+# Check if, e.g. already done on the remote SSH side
 #
-test -z "$SSH_SENDS_LOCALE" || return
+if test -n "$(locale 2>&1 1>/dev/null)"
+then
+    test -n "$LANG" || LANG=C.UTF-8
```

Comment:
this one needs to be -z ie reset LANG in case of error. alternatively could be moved into loop below and just unset. not sure why we need C here

Reply:
There is no POSIX.UTF-8 but C.UTF-8 ... that means multibyte characters within a locale which does ... beside the missed multibyte support ... behaves like POSIX.     This allows to not only see such multibyte character but furthermore to delete and/or move over such multibyte character.

Reply:
Question ... what has to be -z ... the check if the locale program does through an error messages ... or the check for the LANG variable ... IMHO the first case with -n is correct, that is if an error message has been thrown by locale

Reply:
I mean

        test -z "$LANG" || LANG=C.UTF-8

or

         test -z "$LANG" || unset LANG

LANG will be set later then anyway